lua/entities/gmod_wire_expression2/core/globalvars.lua

```diff
@@ -123,9 +125,9 @@ e2function table gtable:toTable()
 		ret.s[index] = v
 		ret.stypes[index] = typeid
 		ret.size = ret.size + 1
-	end
 
-	self.prf = self.prf + ret.size / 3
+		self.prf = self.prf + 3
```

Comment:
Are you aware that you've increased the prf cost of this loop by a factor of 9 with this change?
I'm neither for or against it, just wanna make sure it's a conscious decision since the change does look a bit like it's accidental.

Reply:
I think it's a fair amount considering you really shouldn't be using this function much at all, and removing `ret.size` makes it more lenient if anything